lua/entities/gmod_wire_vectorthruster.lua

```diff
@@ -196,7 +196,7 @@ function ENT:Setup(force, force_min, force_max, oweffect, uweffect, owater, uwat
 	self.lengthismul = lengthismul
 
 	-- Preventing client crashes
-	local BlockedChars = '["?]'
+	local BlockedChars = [=[["?]]=]
```

Comment:
I think this change needs to be undone

Reply:
glualint complains about this

Reply:
tell it to shut the fuck up

Reply:
jk, just revert it.

Reply:
It should actually be '["%?]'

Reply:
It might not complain about `"[\"?]"` but it doesn't really matter